charts/rancher-k3s-upgrader/v0.0.1/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: system-upgrade-controller
+  namespace: cattle-system
+spec:
+  selector:
+    matchLabels:
+      upgrade.cattle.io/controller: system-upgrade-controller
+  template:
+    metadata:
+      labels:
+        upgrade.cattle.io/controller: system-upgrade-controller # necessary to avoid drain
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - {key: "node-role.kubernetes.io/master", operator: In, values: ["true"]}
+      serviceAccountName: system-upgrade
+      tolerations:
+        - key: "node.kubernetes.io/unschedulable" # necessary to avoid cordon/drain
+          operator: "Exists"
+          effect: "NoSchedule"
+      containers:
+        - name: system-upgrade-controller
+          image: rancher/system-upgrade-controller:v0.3.0-rc1
```

Comment:
This can't be hardcoded as it will never get updated if you have a default registry from Rancher (for airgap setups) or tags? 

I expected something like this: https://github.com/rancher/system-charts/blob/dev/charts/rancher-istio/1.4.400/charts/istiocoredns/templates/deployment.yaml#L40